families/track_and_trade/nose2.cfg

```diff
@@ -0,0 +1,8 @@
+[unittest]
```

Comment:
Should this be located at `families/track_and_trade/tests/nose2.cfg`?

Reply:
It's in the right place.